src/chapters/orm/studio.rst

```diff
@@ -0,0 +1,114 @@
+Studio: Creating an ``AbstractEntity``
+======================================
+
+In :ref:`orm1-exercises`, you created a new persistent class, ``EventCategory``. We will eventually use this class to categorize events in our application. However, your application now has a fair amount of duplication across the two model classes. This studio has you reduce repetition (i.e. "DRY out your code") using inheritance and abstract classes.
+
+Getting Ready
+-------------
+
+In your ``coding-events`` repository, checkout out the `add-persistent-category branch <https://github.com/LaunchCodeEducation/coding-events/tree/add-persistent-category>`_ and create a new branch, ``abstract-entity-studio``, off of that branch.
+
+.. admonition:: Note
+
+   You can also branch off of the branch you created in the exercises, if you like. We have provided the branch mentioned to make sure you have good starter code for the studio, even if you didn't complete the exercises. 
+
+Your Tasks
+----------
+
+Let's look at both the ``Event`` and ``EventCategory`` classes. There are four class members that are (essentially) the same in both classes:
+
+- The ``id`` field
+- The ``getId`` method
+- The ``equals`` method
+- The ``hashCode`` method
+
+.. admonition:: Note
+
+   If you have a super-sharp eye, you'll notice that the two ``equals`` methods aren't *exactly* the same. For example, the next-to-last line of this method in ``Event`` looks like this:
+
+   .. sourcecode:: java
+
+      Event Event = (Event) o;
+
+   while in ``EventCategory`` this line is slightly different;
+
+   .. sourcecode:: java
+
+      EventCategory that = (EventCategory) o;
+
+   This minor difference will not matter in the long run, as we'll see later.
+
+Your objective in this studio is to reduce this repetition by creating a class that contains the duplicated/shared code, which both ``Event`` and ``EventCategory`` can inherit.
+
+Create and Extend ``AbstractEntity``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the ``models`` package, create a new class named ``AbstractEntity``. Add the ``abstract`` keyword to the class declaration. We do this because, while we want to share code across our two models, we'll never need to create an instance of ``AbstractEntity``. If you need a quick refresher, review our section on :ref:`abstract-classes`.
+
+.. admonition:: Tip
+
+   This class name seems a little odd at first, but it makes sense after a brief explanation. 
+   
+   It is a commonly-used convention in Java to prefix the name of an abstract class with ``Abstract``, to make it obvious to other developers that they can't create instances of the class. 
```

Comment:
Proof that no hyphen is needed in "commonly-used": https://www.dailywritingtips.com/adverbs-and-hyphens